### PR TITLE
SC-15187 add kubernetes event tags

### PR DESCRIPTION
**Pull request by cpipilas**

### CODE REVIEW

File: docs/tags/common-schema.md [hunk 1]
```diff
@@ -203,6 +203,21 @@ Field Name | Field Type | Required | Notes
 ```creator``` | string       | no       | Person, application, or component that created an event. E.g. ```John Smith```, ```Elasticsearch```, ```Some Batch Job```
 ```data``` | string       | no       | Additional event data. It can be anything you may find useful to have along inside of event object. E.g., it could be  stacktrace in case of ```app_error``` event, base64 encoded content of file, etc.
 
+### Kubernetes Event Tags
+Kubernetes events represent a special type of events that show what's happening inside a cluster, node, pod, or container.
```
Comment (otisg) [inline docs/tags/common-schema.md:207]: Kubernetes events represent a special type of events that show what's happening inside a cluster, node, pod, or container.
=> 
Kubernetes events show what's happening inside a cluster, node, pod, or container.

File: docs/tags/common-schema.md [hunk 2]
```diff
@@ -203,6 +203,21 @@ Field Name | Field Type | Required | Notes
...
+
+Field Name                 | Field Type   | Required | Notes 
+---------------------------|--------------|----------|-------------
+```kubernetes.namespace``` | string       | no       | Namespace represents the namespace of the resource that originated the event
```
Comment (otisg) [inline docs/tags/common-schema.md:211]: Suggest we simplify / trim by dropping:
XXXX represents
XXXX is

File: docs/tags/common-schema.md (hunk 1 above)
Comment (cpipilas) [reply docs/tags/common-schema.md]: Fixed

**Review by otisg — APPROVED** [review]
1 tiny suggestion. LGTM.

File: docs/tags/common-schema.md [hunk 3]
```diff
@@ -203,6 +203,21 @@ Field Name | Field Type | Required | Notes
 ```creator``` | string       | no       | Person, application, or component that created an event. E.g. ```John Smith```, ```Elasticsearch```, ```Some Batch Job```
 ```data``` | string       | no       | Additional event data. It can be anything you may find useful to have along inside of event object. E.g., it could be  stacktrace in case of ```app_error``` event, base64 encoded content of file, etc.
 
+### Kubernetes Event Tags
+Kubernetes events show what's happening inside a cluster, node, pod, or container.
+
+Field Name                 | Field Type   | Required | Notes 
+---------------------------|--------------|----------|-------------
+```kubernetes.namespace``` | string       | no       | Namespace of the resource that originated the event
+```kubernetes.name```      | string       | no       | Name of the resource associated to this event
```
[the remark this answers is not in the document]
Comment (cpipilas) [reply docs/tags/common-schema.md]: Fixed

**Pull request merged**